tests/Tests/ORM/Functional/EnumTest.php

```diff
@@ -470,7 +470,7 @@ public static function provideCardClasses(): iterable
     public function testItAllowsReadingAttributes(): void
     {
         $metadata = $this->_em->getClassMetadata(Card::class);
-        $property = $metadata->getReflectionProperty('suit');
+        $property = $metadata->reflFields['suit'];
```

Comment:
Don't we expect/assert here a deprecation trigger?

Reply:
I didnt pay much attention to this change, but developers downstream (like this test simulates looking at the referenced PR) might want to still access the original reflector.